perceval/rendering/circuit/renderer.py

```diff
@@ -104,7 +104,7 @@ def render_circuit(self,
                     if c.is_composite() and c._components:
                         if recursive:
                             self._current_subblock_info = self._subblock_info.setdefault(c, {})
-                            self.open_subblock(shiftr, c.name, self.get_circuit_size(c, recursive=True), c._color)
+                            self.open_subblock(shiftr, c.name, self.get_circuit_size(c, recursive=False), c._color)
                             self.render_circuit(
```

Comment:
Could you please add a unittest (following our conventions on testing circuit images) using the example from the issue?

Reply:
This is done, let me know if you would like to see other circuit tests.

P.S. This issue seems to be specific to Canvas Renderer. Text Renderer appears to have no problems with box size.

Reply:
Failing tests on Windows does not seem to be related to the proposed changes.